hack/make-rules/vet.sh

```diff
@@ -47,7 +47,9 @@ done
 
 if [[ ${#targets[@]} -eq 0 ]]; then
   # Do not run on third_party directories or generated client code or build tools.
-  targets=$(go list -e ./... | egrep -v "/(build|third_party|vendor|staging|clientset_generated)/")
+  while IFS='' read -r target; do
```

Comment:
is this IFS an empty string?

Reply:
Yes. read() reads one line, so I think splitting by IFS is not necessary on this part.
Do you have any concerns?

Reply:
SGTM